File: Examples/AspNetMvcVB/Models/PossibleViewModel.vb
Imports System.Collections.Generic

Namespace Models

    Public Class PossibleViewModel
        Public Sub New()
            InputFormats = New Dictionary(Of String, List(Of SelectListItem))()
            OutputFormats = New Dictionary(Of String, List(Of SelectListItem))()
        End Sub

        Public Property InputFormatCount As Integer

        Public Property OutputFormatCount As Integer

        Public Property ResultHandlerUrl As String

        Public Property InputFormats As Dictionary(Of String, List(Of SelectListItem))

        Public Property OutputFormats As Dictionary(Of String, List(Of SelectListItem))
    End Class
End NameSpace

File: Examples/AspNetCoreOnNetFullCS/Models/PossibleViewModel.cs
using System.Collections.Generic;
using Microsoft.AspNetCore.Mvc.Rendering;

namespace GleamTech.DocumentUltimateExamples.AspNetCoreOnNetFullCS.Models
{
    public class PossibleViewModel
    {
        public PossibleViewModel()
        {
            InputFormats = new Dictionary<string, List<SelectListItem>>();
            OutputFormats = new Dictionary<string, List<SelectListItem>>();
        }

        public int InputFormatCount { get; set; }

        public int OutputFormatCount { get; set; }

        public string ResultHandlerUrl { get; set; }

        public Dictionary<string, List<SelectListItem>> InputFormats { get; set; }

        public Dictionary<string, List<SelectListItem>> OutputFormats { get; set; }
    }
}
